src/responses/movies.js

```diff
@@ -21,8 +21,7 @@ export const ALREADY_WANTED = tmpl`It looks like ${0} (${1}) is already on your
 /** ${0: Search query} */
 export const NO_MOVIE_FOUND = tmpl`Couldn't find ${0} in your wanted list.`;
 
-export const NO_MOVIE_SLOT = tmpl`Sorry, I couldn't understand the movie title I heard. Please try
-again with different phrasing.`;
+export const NO_MOVIE_SLOT = tmpl`Sorry, I couldn't understand the movie or show title I heard. Please try again with different phrasing.  Please specify show slash series, or movie slash film before the title`;
```

Comment:
I use a max line length of 100 characters, enforced with linting. This is gonna fail ci.